apps/vanilla/src/pages/blackjack/Classes/Player.ts

```diff
@@ -0,0 +1,27 @@
+import { Subscriber } from '../Types/subscriber';
+import { PlayerTurnResult } from '../types/player-turn-result';
+
+import { Inspector } from './inspector';
+import { Result } from './result-displayer';
+
+/** Class illustrate a player in a game.*/
+export class Player extends Inspector implements Subscriber<PlayerTurnResult> {
+	public constructor(private readonly playerIndex: number) {
+		super();
+	}
+
+	/**
+	 * Update turn.
+	 * @param turnResult The result of current dice turn which has result and index of player.
+	 */
+	public update(turnResult: PlayerTurnResult): void {
```

Comment:
```suggestion
  /** @inheritdoc */
	public update(turnResult: PlayerTurnResult): void {
```
Please check all places in the code when they write to you to fix or improve something.